src/sidebar/test/annotation-ui-test.js

```diff
@@ -79,6 +80,19 @@ describe('annotationUI', function () {
         [sinon.match(annot)]);
     });
 
+    it('sets the selectedTab to page if only page notes are present', function () {
```

Comment:
I would suggest we put these tests in the test module that corresponds to the logic changed above, ie. `selections-test.js`. This will mean changing the style of the test a little bit.

I think we also need one more test for the case when annotations are already loaded.

Reply:
I made a separate PR for splitting out the tests. Seems like it might bet better to just split out all these all at once rather than doing it a little at a time. I'll add a test for where annotations are already loaded-that's definitely important.